<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-JONTFOMF" failures="1" tests="4" name="Test" time="10.008" errors="0" timestamp="2024-11-09T09:24:44 IST">
  <testcase classname="api.test.UserTests_ByUsingPropFile" name="testPostUser" time="3.361"/>
  <testcase classname="api.test.UserTests_ByUsingPropFile" name="testGetUserbyName" time="2.142"/>
  <testcase classname="api.test.UserTests_ByUsingPropFile" name="testUpdateUserbyName" time="2.036">
    <failure type="java.lang.AssertionError" message="expected [200] but found [405]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at api.test.UserTests_ByUsingPropFile.testUpdateUserbyName(UserTests_ByUsingPropFile.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- testUpdateUserbyName -->
  <testcase classname="api.test.UserTests_ByUsingPropFile" name="testDeleteUserbyName" time="1.634"/>
</testsuite> <!-- Test -->
